front_end/html/edit_exercise.html

```diff
@@ -226,44 +231,31 @@ <h2>Create exercise</h2>
         </div>
 
         <div class="shadow-box">
+            <div id="test_upload">
+                <p><strong>Tests:</strong></p>
+                <p>Test code can be used to run one or more tests using the student's code. It helps test the integrity of their code rather than its ability to produce the expected output.</p>
```

Comment:
Please change to "You can write one or more tests that evaluate the integrity of the student's code. Oftentimes, tests are used when the instructor has asked students to write a function, and each test invokes the function using certain parameters. Either way, the test code will be invoked *after* the student's code. For each test, you may indicate whether the test code should be visible to students. Hidden tests help to ensure that the student cannot guess the expected output  Generally, we recommend including at least one visible test and one hidden test. We recommend that hidden tests use similar approaches as visible tests."

Reply:
Also, I forgot whether we are adding the ability for students to receive partial credit if they pass some tests but not others?

Reply:
We haven't added that, but I could if you'd like!

Reply:
Please remind me that we should discuss this as well in our next meeting.

Reply:
I'm afraid I forgot to remind you, but I will on Thursday!